build.gradle

```diff
@@ -178,7 +155,7 @@ configurations {
     configureEach {
         resolutionStrategy {
             dependencySubstitution {
-                substitute module('org.apache.commons:commons-compress') using module('org.apache.commons:commons-compress:1.26.2')
+                substitute module('org.apache.commons:commons-compress') using module("${libs.commons.compress.get()}")
             }
         }
     }
```

Comment:
question: do we still need those constrains? can we update the version of testcontainers library?